mne/io/eeglab/eeglab.py

```diff
@@ -450,25 +450,33 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         if events is None and eeg.trials > 1:
             # first extract the events and construct an event_id dict
             event_name, event_latencies, unique_ev = list(), list(), list()
-            ev_idx = 0
             warn_multiple_events = False
             epochs = _bunchify(eeg.epoch)
             events = _bunchify(eeg.event)
+
+            # construct urevent idx -> event idx mapping
+            ur2ev = {evnt.urevent: idx for idx, evnt in enumerate(events)}
```

Comment:
remind me, what does `urevent` mean?

Reply:
its field in the event file. ok.

Reply:
yes, it is a field containing unique event identifier.